crypto/rand/drbg_lib.c

```diff
@@ -656,6 +668,73 @@ int RAND_DRBG_set_reseed_time_interval(RAND_DRBG *drbg, time_t interval)
     return 1;
 }
 
+
+/*
+ * Locks the given drbg
```

Comment:
You can make this three sentences and word-wrap the comment.  Side-note.

Reply:
Wrapped two of them. Hope I understood you right.